source/tests/yats-test/scheduler_test.cpp

```diff
@@ -25,10 +29,27 @@ TEST(scheduler_test, multithreaded_timing_test)
     auto end = std::chrono::high_resolution_clock::now();
 
     auto duration = std::chrono::duration_cast<std::chrono::milliseconds>(end - start).count();
+
     EXPECT_GE(duration, 100);
     EXPECT_LE(duration, 500);
 }
 
+TEST(scheduler_test, run_twice)
+{
+    yats::pipeline pipeline;
```

Comment:
Since we are using the yats namespace in tests, please remove the "yats::" identifier in tests.